src/main/java/org/opentripplanner/routing/vehicle_rental/VehicleRentalSystem.java

```diff
@@ -18,7 +16,6 @@ public class VehicleRentalSystem {
   public final String phoneNumber;
   public final String email;
   public final String feedContactEmail;
-  public final ZoneId timezone;
```

Comment:
Would it be ok to just store it as String here?

Reply:
Sure, why not.